.github/ISSUE_TEMPLATE/DesignDoc.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Design Doc tracking issue
+about: A standard template to follow when leading a project
+labels: Daily, NewFeature
+---
+
+## Proposal
+
+## Tasks
+
+- [ ] Post **Proposal (full Problem/Solution statement)** in `#expensify-open-source`
+- [ ] Wait at least one full business day, and until the post has a majority (2/3) of positive reactions (👍)
+- [ ] Paste Proposal in the space above with a link to the Slack thread
+- [ ] Email `strategy@expensify.com` and paste in the Proposal
+- [ ] Fill out the **High-level overview of the problem**, **Timeline**, and **Terminology** sections of the Design Doc
```

Comment:
Should we specify here that we want them to do the work _in the doc_ as opposed to somewhere else and then copy/pasting? (That's something that I did when I first joined Expensify and it causes confusion)

Reply:
Oh hmm - I think we can see if it's a problem? I still do this to this day as I am self conscious about people reading my random nonsense drafts 😄